sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -356,8 +385,8 @@ Administrators who desire increased control over external collaborators should c
 Note, this only applies to new external users accepting new sharing invitations. Also, the resource owner must share with an organizational or Microsoft account or the external user will be unable to access the resource.
 
 The valid values are:  
-False (default) - When a document is shared with an external user, bob@contoso.com, it can be accepted by any user with access to the invitation link in the original e-mail.  
-True - User must accept this invitation with bob@contoso.com.
+- False (default) - When a document is shared with an external user, bob@contoso.com, it can be accepted by any user with access to the invitation link in the original e-mail.  
```

Comment:
```suggestion
- False (default) - When a document is shared with an external user, bob@contoso.com, it can be accepted by any user with access to the invitation link in the original email.  
```